py-rse/teams.Rmd

```diff
@@ -445,17 +468,22 @@ You can't force new issues to be as complete as you might like,
 but you can use an issue template to make it easier for contributors to
 remember and complete documentation about bug reports.
 
-## How can we use labels to organize work? {#teams-labels}
+FIXME: how to respond to a bug report.
+Sometimes the person creating the issue may not know or have the right answer for some of these things,
+and will be doing their best with limited information about the error.
+Responding with kindness and encouragement is important to maintain a healthy community,
+and should be enforced by the project's Code of Conduct (Section \@ref(#teams-coc)).
+
+## Labeling Issues {#teams-labels}
 
-There is always more work to do than there is time to do it.
-Issue trackers let project members add [labels][issue-label] to issue to manage this.
+Issue trackers let project members add [labels][issue-label] to issues
+to make things easier to search and organize.
 Labels are also often called [tags][tag];
 whatever term is used,
-they are just a word or two to help users sort and search among tasks.
+each one is just a descriptive word or two.
 
-GitHub allows project owners to create any labels they want
-that users can then add to their issues.
-A small project should always use some version of these three labels:
+GitHub allows project owners to define any labels they want.
```

Comment:
Once we've created a bug report/issue in Amira's repo (see previous comment) we could screenshot the drop down labels menu to illustrate what we're talking about in this labels section? e.g. 
<img width="1010" alt="Screen Shot 2020-06-23 at 1 41 02 pm" src="https://user-images.githubusercontent.com/2062210/85358532-56b97b80-b557-11ea-983b-38ea7791e0af.png">

Reply:
Added to issue #405.